src/main/java/reposense/model/RepoLocation.java

```diff
@@ -61,30 +67,43 @@ public RepoLocation(String location) throws InvalidLocationException {
         }
 
         this.location = location;
-        String[] repoNameAndOrg;
+        String[] remoteRepoNameAndOrg;
+        String[] outputFolderRepoNameAndOrg;
         if (location.isEmpty()) {
-            repoNameAndOrg = new String[] {"", "", UNSUPPORTED_DOMAIN_NAME};
+            remoteRepoNameAndOrg = new String[] {"", "", UNSUPPORTED_DOMAIN_NAME};
+            outputFolderRepoNameAndOrg = remoteRepoNameAndOrg;
         } else if (isLocalRepo(location)) {
-            repoNameAndOrg = getLocalRepoNameAndOrg(location);
+            outputFolderRepoNameAndOrg = getLocalRepoNameAndOrg(location);
```

Comment:
Nothing to do with this line in particular, but are you also able to update the Javadoc for this `getLocalRepoNameAndOrg()` function? It is still referring to the array returned as length 2. Same for the `getRemoteRepoNameAndOrg()` function.